src/MPDevice.cpp

```diff
@@ -6402,12 +6410,18 @@ void MPDevice::getStoredFiles(std::function<void (bool, QStringList)> cb)
         Q_UNUSED(data);
 
         /* List constructor */
-        QStringList list = QStringList();
+        QList<QVariantMap> list;
 
         /* Get file names */
         for (auto &i: dataNodes)
         {
-            list.append(i->getService());
+            QVariantMap item;
+            item.insert("name", i->getService());
+
+            // FIXME: This isn't the actual file size we should get the real one.
+            item.insert("size", i->getDataNodeData().length());
```

Comment:
Heads up here, this isn't the real file size. It should be replaced with the real one.
I was told that someone else will do it so I didn't go forward.

Reply:
we'll talk about that.